src/components/Footer.jsx

```
@@ -0,0 +1,141 @@
+
+const SocialButton = styled(IconButton)(({ theme }) => ({
+  color: "#fff",
+  "&:hover": {
+    backgroundColor: theme.palette.secondary.main,
+    transform: "scale(1.1)"
+  }
+}));
+
+const Footer = () => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```

ESLint

[error] 'isMobile' is assigned a value but never used.

(no-unused-vars)